src/style.css

```diff
@@ -1,3 +1,79 @@
+html,
 body {
+  padding: 0;
   margin: 0;
+  width: 100%;
+}
+
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+:root {
+  --primary-color: #00acdc;
+}
+
+header {
+  justify-content: space-between;
+  display: flex;
+  background: #fff;
+  align-items: center;
+  padding: 0 50px;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. Instead of styling the `header` tag directly, assign a class to it and apply the styles to that class.